ofl/hachimarupop/upstream.yaml

```diff
@@ -3,4 +3,3 @@ files:
   fonts/ttf/HachiMaruPop-Regular.ttf: HachiMaruPop-Regular.ttf
   OFL.txt: OFL.txt
   DESCRIPTION.en_us.html: DESCRIPTION.en_us.html
-repository_url: https://github.com/noriokanisawa/HachiMaruPop.git
```

Comment:
@aaronbell I think we want to keep the repo urls.

Reply:
Oh! Sure. Don’t know why the packager removed them. Feel free to disregard that change.